code-of-conduct.md

```diff
@@ -0,0 +1,23 @@
+# Code of Conduct
+
+## Introduction
+
+Biomakespace is building a multidisciplinary, volunteer-driven community laboratory for collaborative experimentation at the interface of biology and engineering. We aim to provide a supportive, safe environment for discovery and innovation through team-based projects and curiosity-driven research.
```

Comment:
It may be good to elaborate more on topics of openness and equity.

Reply:
👍